source/administration/minio-console.rst

```diff
@@ -137,416 +118,75 @@ routing rules may require setting a static MinIO Console port. For example,
-
-You can select an existing notification target from the list to view its details or delete the target.
-
-.. image:: /images/minio-console/console-add-notification-target.png
-   :width: 600px
-   :alt: The MinIO Console's Notification screen after selecting add new target that shows the types of destination targets users can add.
-   :align: center
-
-Tiers
------
-
-.. image:: /images/minio-console/console-settings-tiers.png
-   :width: 600px
-   :alt: MinIO Console Settings - Tiering
-   :align: center
-
-The :guilabel:`Tiers` section provides an interface for adding and managing :ref:`remote tiers <minio-lifecycle-management-tiering>` to support lifecycle management transition rules.
-
-Select the :guilabel:`Create Tier +` button to add a new tier to the deployment.
-Choose to add a MinIO, Google Cloud Storage, AWS S3, or Azure tier type.
+Logging into the MinIO Console depends on how you configured identity management for the deployment.
 
-Existing tiers display with details of their configuration and an icon showing their current online or offline status.
+- When using the built-in MinIO identity management solution, the sign-in screen displays a standard login screen.
+  Enter your Username and Password to log in to the MinIO Console.
+- If logging in with a third party application and :ref:`MinIO's Security Token Service (STS) <minio-security-token-service>`, select :guilabel:`Use STS` and enter the Username, Secret, and Token.
+- If the deployment uses a single OpenID or Active Directory/LDAP identity provider solution, select the provider's button to proceed to the login screen.
+- If the deployment has multiple OpenID and/or Active Directory/LDAP identify management providers configured, the MinIO Console's sign-in screen provides a dropdown list of providers.
+  Select the provider you wish to use to log in to the MinIO Console, then enter the credentials.
 
-You can select an existing tier from the list to view its details or make changes.
+.. admonition:: Try out the Console using MinIO's Play testing environment
+   :class: note
 
-Site Replication
-----------------
+   You can explore the Console using https://play.min.io:9443. 
+   Log in with the following credentials:
 
-.. image:: /images/minio-console/console-settings-site-replication.png
-   :width: 600px
-   :alt: MinIO Console Settings - Site Replication
-   :align: center
+   - Username: ``Q3AM3UQ867SPQQA43P2F``
+   - Password: ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG``
 
-The :guilabel:`Site Replication` section provides an interface for adding and managing the site replication configuration for the deployment.
+   The Play Console connects to the MinIO Play deployment at https://play.min.io.
+   You can also access this deployment using :mc:`mc` and using the ``play`` alias.
 
-Configuring site replication requires that only a single site have existing buckets or objects (if any).
+.. _minio-console-settings:
 
 Settings
```

Comment:
We can probably drop this section now right?

Reply:
Maybe? It's not mentioned anywhere else.

Reply:
Seems to fit more with the configuration on this page than on managing deployments.

Reply:
ON further consideration, moved to the managing deployments page.